dev/bots/run_fuchsia_tests.sh

```diff
@@ -38,9 +42,24 @@ else
 fi
 
 reboot() {
-  # note: this will set an exit code of 255, which we can ignore.
+  $script_dir/fuchsia_ctl -d $device_name ssh \
```

Comment:
@godofredoc do you know if fuchsia_ctl needs the flags to be ordered this way? It makes it hard to read to have the command mixed in with the flags.

Reply:
Yes, -d is an option for the core command and the ones after "ssh" applies only to the ssh command.